modules/apps/account/account-service/src/main/java/com/liferay/account/service/impl/AccountEntryLocalServiceImpl.java

```diff
@@ -681,6 +714,64 @@ public AccountEntry updateAccountEntry(
 		return accountEntry;
 	}
 
+	@Override
+	public AccountEntry updateAddresses(
```

Comment:
this is actually adding, updating and deleting addresses, I'm not sure this is the right name, but I wonder what could be the right name for this method :O

Reply:
it's the same name as in UsersAdminUtil (with mostly the same code). I'm not able to use directly this Util because it's using for example `AddressServiceUtil.addAddress` that is checking for a different permission (UPDATE) instead of MANAGE_ADDRESS like I need